config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-release-1-2.yaml

```diff
@@ -0,0 +1,277 @@
+presubmits:
+  kubernetes-sigs/cluster-api:
+  - name: pull-cluster-api-build-release-1-2
+    decorate: true
+    path_alias: sigs.k8s.io/cluster-api
+    always_run: true
+    labels:
+      preset-service-account: "true"
+    branches:
+    # The script this job runs is not in all branches.
+    - ^release-1.2$
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220614-cde7860a33-1.24
+        command:
+        - runner.sh
+        - ./scripts/ci-build.sh
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-cluster-api
```

Comment:
dashboard must be adjusted as well (please diff against the release-1.1 presubmits for that diff)

Same for other files